cloud/cloud.go

```diff
@@ -119,9 +119,13 @@ const sentinelFilePath = "/tmp/.wr_cloud_sentinel"
 var sentinelInitScript = []byte("#!/bin/bash\nsed -i 's/^Defaults\\s*requiretty/Defaults\\t!requiretty/' /etc/sudoers\nsed -i '/user_allow_other/s/^#//g' /etc/fuse.conf\nchmod o+r /etc/fuse.conf\ntouch " + sentinelFilePath)
 
 // sentinelTimeOut is how long we wait for sentinelFilePath to be created before
-// we give up and return an error from Spawn().
+// we give up and return an error from WaitUntilReady().
 var sentinelTimeOut = 10 * time.Minute
 
+// pcsTimeOut is how long we wait for a user's post creation script to exit
+// before we give up and return an error from WaitUntilReady().
+var pcsTimeOut = 15 * time.Minute
```

Comment:
Probably good idea to mention the 15 min limit in docs. Something like "hard limit of 15 min execution"

Reply:
Thanks. Documentation added.